docs/aks/onboarding-form.md

```diff
@@ -0,0 +1,45 @@
+# Teacher services infra - Service onboarding form
+
+Note - this is a migration of two existing services. They share a single codebase, are served to the public from
+two different domains, and are currently deployed to (non-AKS) Azure.
+
+- Service name? Access your teaching qualifications / Check a teacher's record (two services, single codebase)
+- Domains ?
+  - access-your-teaching-qualifications.education.gov.uk (already exists)
+  - check-a-teachers-record.education.gov.uk (already exists)
+  - access-your-teaching-qualifications.teacherservices.cloud
+  - check-a-teachers-record.teacherservices.cloud
+  - test._ and preprod._ variants of the above.
+- Service short name? aytqctr
```

Comment:
The existing service short name is just aytq — using aytqctr here so it's a little more obvious that this app consists of two services.

Reply:
I think it's limited to 6 characters. Maybe atqctr? or tqtr? or tqr?

Reply:
Let's keep aytq 👍🏼